docs/en/source/index.rst

```diff
@@ -1,3 +1,17 @@
+=====================
+Scorpio Broker
+=====================
+
+Scorpio broker implements the NGSI APIs through which the producers and consumers can interact with each other. For Example in the typical IoT based room, various sensors like temperature sensors, light sensors, etc are connected to the central application which uses those sensors output and acts as the consumer. There can be a lot of use cases for this central application i.e Scorpio. 
+
+1. Scorpio doesn't encode the data for storing so it can be used as a data lake.
+
+2. Scorpio provides several interfaces for querying the stored data so easily analytics can be done on the stored data. like it can be used to predict the situation of an ecosystem. Example:- In a huge building there can be several fire sensors, temperature sensors, and smoke sensors. In case of a false fire alarm, it can be verified by the collected fire data, temperature data and smoke data of the particular area. 
+
+3. Scorpio can be used for determining the accuracy of any event. For example, In an automated car, the speed of the car can be known by several applications like GPS, speed camera and speedometer. Scorpio's internal data is stored in this way that any third-party application can use it and can find the accuracy and determine faulty device.
```

Comment:
I feel like this needs some rewriting further down the road. You of course could detect a faulty device based on the values but that's not the main point of NGSI-LD. Device health is a bit out of scope for it. It's more about the highlevel modelling that you can easy look for high level entities, e.g. cars, and can find all attributes associated to that. 
@martin-p-bauer do you have a good sentence describing that?